### DISCUSSION TITLE
k8s容器doris中添加paimon catalog未生效

### DISCUSSION THREAD

**Discussion by xiaolin84250** [apache/doris#25284] [A - General / Q&A] [answered]
jar的路径和文件是通过pvc挂载的，并且尝试过重启，还是没能生效，0.4版本的jar也尝试过，doris的版本是2.0.0的

![image](https://github.com/apache/doris/assets/38107489/ea3686bd-d9cc-4646-b0fd-e8082a7a8439)

![image](https://github.com/apache/doris/assets/38107489/fcea07c3-b58c-43ff-9978-000c1d83f3ae)

**Comment by LemonLiTree** [comment]
which doris version?

**Reply by xiaolin84250** [reply]
[2.0.0](doris:2.0.0_alpha-be-x86_64)
[2.0.0](doris:2.0.0_alpha-fe-x86_64)

**Comment by ixzc** [comment]
refer this：https://doris.apache.org/zh-CN/docs/dev/lakehouse/multi-catalog/paimon
and you'd better upgrade doris to version 2.0.2

**Reply by xiaolin84250** [reply]
I refer to this tutorial. The docker image of version 2.0.2 has not been updated yet. The latest version is version 2.0.0.

**Reply by ixzc** [reply]
<img width="525" alt="image" src="https://github.com/apache/doris/assets/49646212/5a5e5828-ec45-48c8-8ade-4ae3bc087c0b">
it worked？

**Reply by xiaolin84250** [reply]
This is no problem if I create the hive catalog separately, but the catalog I created for paimon shows that it does not exist. I have put the jar into the specified location and tried to restart.

**Reply by xiaolin84250** [reply]
![image](https://github.com/apache/doris/assets/38107489/36a05241-f15b-484c-81c1-de1feac6c5b0)

**Reply by ixzc** [reply, marked as answer]
This format has been supported since 2.0.1
You can find the image of 2.0.2 from the link : https://hub.docker.com/r/selectdb/

**Reply by xiaolin84250** [reply]
Okay, I'll try it, thank you
